tests/CommonTestUtils/GitModuleTestHelper.Cleanup.cs

```diff
@@ -0,0 +1,78 @@
+using GitUI;
+using Microsoft.VisualStudio.Threading;
+
+namespace CommonTestUtils;
+
+public partial class GitModuleTestHelper : IDisposable
+{
+#if CI_BUILD
+    static GitModuleTestHelper()
+    {
+        Console.WriteLine("GitModuleTestHelper: Disabling explicit clean-up for continuous integration test environment");
+    }
+
+    private void FileAndForgetCleanUp()
+    {
+    }
+
+    public static void WaitForCleanUpCompletion()
+    {
+    }
+#else
+    static GitModuleTestHelper()
+    {
+        Console.WriteLine("GitModuleTestHelper: Will perform clean-up in background tasks");
+
+        Epilogue.RegisterAfterSuiteAction(order: 2, WaitForCleanUpCompletion);
+    }
+
+    private static TaskManager CleanUpOperations = new(new JoinableTaskContext());
```

Comment:
Please move fields to the top, and

```suggestion
    private static TaskManager _cleanUpOperations = new(new JoinableTaskContext());
```